js/request.jsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { render } from 'react-dom';
 import { Router, Route, Switch } from 'react-router-dom';
 import createBrowserHistory from 'history/createBrowserHistory';
-
```

Comment:
Nitpick, I prefer to separate out imports into three groups, core (fs, os, path, etc), package dependencies (react-dom), and then relative app dependencies. Each separated by a blank line. That's just me :)